DevelopmentProposals/template.md

```diff
@@ -0,0 +1,61 @@
+## Project: TITLE
+***OpenSim Design Document***
+
+### Overview
+
+Just a few sentences that describes the purpose of this project. 
```

Comment:
How about changing line 6 to the following: 
Describe the motivation for the change, by (1) including issue reference(s) or (2) listing benefits, ideally to users.